scripts/optimize_teaching.sh

```
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+if [ -f .env ]; then
+    export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)